### PR TITLE
Flush content whenever its size reaches the buffer max limit

### PR DESCRIPTION
**Pull request by yykamei**
Currently, the trace of the analysis is flushed at the end of the
analysis. However, I want to see the on-going trace log even though the
analysis has not yet finished. Also, I want to know whether the analysis
has started or not, so at the beginning of the analysis, I inserted the
`io.flush!` procedure.

### CODE REVIEW

File: lib/runners/cli.rb [hunk 1]
```diff
@@ -38,6 +38,7 @@ def processor_class
     end
 
     def run
+      io.finalize! # Write or upload empty content to let the caller of Runners notice the beginning of the analysis.
```
Comment (ybiquitous) [inline lib/runners/cli.rb]: [IMO] `finalize!` seems inaccurate to me... (not final)

Comment (yykamei) [reply lib/runners/cli.rb]: I agree. How about adding `alias flush! finalize!`?

Comment (ybiquitous) [reply lib/runners/cli.rb]: Adding alias seems confusing to me? How about renaming `finalize!` to `flush!`?

Comment (yykamei) [reply lib/runners/cli.rb]: Good :+1: I will fix it like that.

**Pull request merged**